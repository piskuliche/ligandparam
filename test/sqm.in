Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   6    C1       -1.9500        5.9300        0.3080 
   6    C2       -1.1330        6.0490       -0.9920 
   1    H1       -1.3900        6.3440        1.1210 
   1    H2       -2.8710        6.4650        0.2010 
   1    H3       -0.2120        5.5140       -0.8840 
   1    H4       -1.6930        5.6360       -1.8040 
   6    C3       -0.8340        7.5330       -1.2760 
   8    O1        0.2120        8.0630       -0.8190 
   7    N1       -1.7690        8.3290       -2.0830 
   1    H5       -2.4480        8.7540       -1.4830 
   6    C4        0.2990        9.1600       -3.1840 
   6    C5        1.0120       10.1550       -3.8720 
   6    C6       -1.0280        9.3750       -2.8020 
   6    C7        2.3200        9.8900       -4.2260 
   6    C8        0.4030       11.3580       -4.1770 
   6    C9       -1.6460       10.5880       -3.1080 
   1    H6        2.3300        9.0300       -4.8610 
   6   C10        1.0510       12.3890       -4.8570 
   6   C11       -0.9310       11.5760       -3.7940 
   1    H7       -2.6610       10.7620       -2.8200 
   6   C12        2.3980       12.1220       -5.2230 
   1    H8        3.8600       10.7490       -5.1500 
   1    H9        0.5010       12.5870       -5.7530 
   1   H10       -1.4050       12.5070       -4.0280 
   1   H11        3.0080       12.8670       -4.7560 
   1   H12        0.7750        8.2310       -2.9510 
   7    N2        2.9110       10.9070       -4.8790 
   1   H13        1.0490       13.2510       -4.2230 
   1   H14        2.8770        9.6930       -3.3340 
   1   H15        2.4590       12.2010       -6.2880 
   6   C13       -2.2490        4.4470        0.5930 
   6   C14       -1.3480        3.6840        1.3360 
   6   C15       -3.4210        3.8660        0.1080 
   6   C16       -1.6190        2.3400        1.5930 
   1   H16       -0.4240        4.1420        1.7170 
   6   C17       -3.6920        2.5220        0.3660 
   1   H17       -4.1320        4.4670       -0.4770 
   6   C18       -2.7910        1.7590        1.1090 
   1   H18       -4.6170        2.0640       -0.0160 
   1   H19       -3.0040        0.6990        1.3120 
   9    F1       -0.7460        1.6010        2.3110 

