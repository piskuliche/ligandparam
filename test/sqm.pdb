ATOM      1  C1  FBK     1      -1.644   5.756  -0.068  1.00  0.00           C
ATOM      2  C2  FBK     1      -1.028   5.388  -1.405  1.00  0.00           C
ATOM      3  H1  FBK     1      -0.846   6.169   0.607  1.00  0.00           H
ATOM      4  H2  FBK     1      -2.412   6.563  -0.208  1.00  0.00           H
ATOM      5  H3  FBK     1      -0.304   4.541  -1.269  1.00  0.00           H
ATOM      6  H4  FBK     1      -1.821   5.048  -2.121  1.00  0.00           H
ATOM      7  C3  FBK     1      -0.252   6.557  -1.995  1.00  0.00           C
ATOM      8  O1  FBK     1       0.990   6.521  -2.060  1.00  0.00           O
ATOM      9  N1  FBK     1      -1.010   7.625  -2.451  1.00  0.00           N
ATOM     10  H5  FBK     1      -1.997   7.560  -2.332  1.00  0.00           H
ATOM     11  C4  FBK     1       0.797   9.049  -3.361  1.00  0.00           C
ATOM     12  C5  FBK     1       1.187  10.258  -3.942  1.00  0.00           C
ATOM     13  C6  FBK     1      -0.552   8.820  -3.030  1.00  0.00           C
ATOM     14  C7  FBK     1       2.643  10.448  -4.255  1.00  0.00           C
ATOM     15  C8  FBK     1       0.246  11.261  -4.206  1.00  0.00           C
ATOM     16  C9  FBK     1      -1.499   9.841  -3.300  1.00  0.00           C
ATOM     17  H6  FBK     1       2.994   9.573  -4.879  1.00  0.00           H
ATOM     18  C10 FBK     1       0.657  12.535  -4.848  1.00  0.00           C
ATOM     19  C11 FBK     1      -1.095  11.035  -3.879  1.00  0.00           C
ATOM     20  H7  FBK     1      -2.562   9.697  -3.056  1.00  0.00           H
ATOM     21  C12 FBK     1       2.154  12.799  -4.684  1.00  0.00           C
ATOM     22  H8  FBK     1       3.937  11.864  -4.891  1.00  0.00           H
ATOM     23  H9  FBK     1       0.401  12.489  -5.941  1.00  0.00           H
ATOM     24  H10 FBK     1      -1.842  11.818  -4.084  1.00  0.00           H
ATOM     25  H11 FBK     1       2.335  13.164  -3.630  1.00  0.00           H
ATOM     26  H12 FBK     1       1.563   8.279  -3.167  1.00  0.00           H
ATOM     27  N2  FBK     1       2.965  11.644  -5.009  1.00  0.00           N
ATOM     28  H13 FBK     1       0.093  13.397  -4.403  1.00  0.00           H
ATOM     29  H14 FBK     1       3.203  10.404  -3.272  1.00  0.00           H
ATOM     30  H15 FBK     1       2.455  13.628  -5.385  1.00  0.00           H
ATOM     31  C13 FBK     1      -2.270   4.570   0.582  1.00  0.00           C
ATOM     32  C14 FBK     1      -1.459   3.565   1.116  1.00  0.00           C
ATOM     33  C15 FBK     1      -3.662   4.463   0.673  1.00  0.00           C
ATOM     34  C16 FBK     1      -2.051   2.453   1.739  1.00  0.00           C
ATOM     35  H16 FBK     1      -0.363   3.638   1.055  1.00  0.00           H
ATOM     36  C17 FBK     1      -4.243   3.357   1.293  1.00  0.00           C
ATOM     37  H17 FBK     1      -4.302   5.255   0.257  1.00  0.00           H
ATOM     38  C18 FBK     1      -3.451   2.346   1.828  1.00  0.00           C
ATOM     39  H18 FBK     1      -5.339   3.282   1.361  1.00  0.00           H
ATOM     40  H19 FBK     1      -3.902   1.472   2.318  1.00  0.00           H
ATOM     41  F1  FBK     1      -1.267   1.477   2.257  1.00  0.00           F
