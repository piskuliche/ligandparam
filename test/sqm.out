            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 57

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: F : M.J.S.DEWAR et al. THEOCHEM, 180, 1, (1988)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =      107
| QMMM:    diag iterations used for timing =       10
| QMMM:
| QMMM:              Internal diag routine = 0.012978 seconds
| QMMM:                 Dspev diag routine = 0.012075 seconds
| QMMM:                Dspevd diag routine = 0.009029 seconds
| QMMM:                Dspevx diag routine = 0.047848 seconds
| QMMM:                 Dsyev diag routine = 0.013335 seconds
| QMMM:                Dsyevd diag routine = 0.010380 seconds
| QMMM:                Dsyevr diag routine = 0.011878 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.005309 seconds
| QMMM:
| QMMM: Using dspevd routine (diag_routine=3).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C       -1.9500    5.9300    0.3080
  QMMM:     2        2      C       -1.1330    6.0490   -0.9920
  QMMM:     3        3      H       -1.3900    6.3440    1.1210
  QMMM:     4        4      H       -2.8710    6.4650    0.2010
  QMMM:     5        5      H       -0.2120    5.5140   -0.8840
  QMMM:     6        6      H       -1.6930    5.6360   -1.8040
  QMMM:     7        7      C       -0.8340    7.5330   -1.2760
  QMMM:     8        8      O        0.2120    8.0630   -0.8190
  QMMM:     9        9      N       -1.7690    8.3290   -2.0830
  QMMM:    10       10      H       -2.4480    8.7540   -1.4830
  QMMM:    11       11      C        0.2990    9.1600   -3.1840
  QMMM:    12       12      C        1.0120   10.1550   -3.8720
  QMMM:    13       13      C       -1.0280    9.3750   -2.8020
  QMMM:    14       14      C        2.3200    9.8900   -4.2260
  QMMM:    15       15      C        0.4030   11.3580   -4.1770
  QMMM:    16       16      C       -1.6460   10.5880   -3.1080
  QMMM:    17       17      H        2.3300    9.0300   -4.8610
  QMMM:    18       18      C        1.0510   12.3890   -4.8570
  QMMM:    19       19      C       -0.9310   11.5760   -3.7940
  QMMM:    20       20      H       -2.6610   10.7620   -2.8200
  QMMM:    21       21      C        2.3980   12.1220   -5.2230
  QMMM:    22       22      H        3.8600   10.7490   -5.1500
  QMMM:    23       23      H        0.5010   12.5870   -5.7530
  QMMM:    24       24      H       -1.4050   12.5070   -4.0280
  QMMM:    25       25      H        3.0080   12.8670   -4.7560
  QMMM:    26       26      H        0.7750    8.2310   -2.9510
  QMMM:    27       27      N        2.9110   10.9070   -4.8790
  QMMM:    28       28      H        1.0490   13.2510   -4.2230
  QMMM:    29       29      H        2.8770    9.6930   -3.3340
  QMMM:    30       30      H        2.4590   12.2010   -6.2880
  QMMM:    31       31      C       -2.2490    4.4470    0.5930
  QMMM:    32       32      C       -1.3480    3.6840    1.3360
  QMMM:    33       33      C       -3.4210    3.8660    0.1080
  QMMM:    34       34      C       -1.6190    2.3400    1.5930
  QMMM:    35       35      H       -0.4240    4.1420    1.7170
  QMMM:    36       36      C       -3.6920    2.5220    0.3660
  QMMM:    37       37      H       -4.1320    4.4670   -0.4770
  QMMM:    38       38      C       -2.7910    1.7590    1.1090
  QMMM:    39       39      H       -4.6170    2.0640   -0.0160
  QMMM:    40       40      H       -3.0040    0.6990    1.3120
  QMMM:    41       41      F       -0.7460    1.6010    2.3110

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10      -36.2159 kcal/mol        1.0474 kcal/(mol*A)
xmin    20      -39.5453 kcal/mol        1.0151 kcal/(mol*A)
xmin    30      -40.6082 kcal/mol        0.7025 kcal/(mol*A)
xmin    40      -41.5245 kcal/mol        0.3935 kcal/(mol*A)
xmin    50      -42.0680 kcal/mol        1.4001 kcal/(mol*A)
xmin    60      -42.4460 kcal/mol        0.3856 kcal/(mol*A)
xmin    70      -42.4760 kcal/mol        0.2101 kcal/(mol*A)
xmin    80      -42.4025 kcal/mol        1.3122 kcal/(mol*A)
xmin    90      -42.4815 kcal/mol        0.0794 kcal/(mol*A)
xmin   100      -42.4881 kcal/mol        0.0249 kcal/(mol*A)
xmin   110      -42.5025 kcal/mol        0.0963 kcal/(mol*A)
xmin   120      -42.5083 kcal/mol        0.0822 kcal/(mol*A)
xmin   130      -42.5269 kcal/mol        0.0872 kcal/(mol*A)
xmin   140      -42.5434 kcal/mol        0.0820 kcal/(mol*A)
xmin   150      -42.5474 kcal/mol        0.0500 kcal/(mol*A)
xmin   160      -42.5516 kcal/mol        0.0732 kcal/(mol*A)
xmin   170      -42.5551 kcal/mol        0.0853 kcal/(mol*A)
xmin   180      -42.5573 kcal/mol        0.0180 kcal/(mol*A)
xmin   190      -42.5597 kcal/mol        0.0228 kcal/(mol*A)
xmin   200      -42.5604 kcal/mol        0.0263 kcal/(mol*A)
xmin   210      -42.5607 kcal/mol        0.0504 kcal/(mol*A)
xmin   220      -42.5423 kcal/mol        0.3806 kcal/(mol*A)
xmin   230      -42.5610 kcal/mol        0.0791 kcal/(mol*A)
xmin   240      -42.5621 kcal/mol        0.0187 kcal/(mol*A)
xmin   250      -42.5626 kcal/mol        0.0028 kcal/(mol*A)
xmin   260      -42.5626 kcal/mol        0.0154 kcal/(mol*A)
xmin   270      -42.5627 kcal/mol        0.0058 kcal/(mol*A)
xmin   280      -42.5627 kcal/mol        0.0040 kcal/(mol*A)
xmin   290      -42.5628 kcal/mol        0.0043 kcal/(mol*A)
xmin   300      -42.5629 kcal/mol        0.0027 kcal/(mol*A)
